packages/fxa-payments-server/src/components/PlanDetails/index.tsx

```diff
@@ -207,7 +207,7 @@ export const PlanDetails = ({
 
           {!detailsHidden && productDetails.details && (
             <div
-              className="mt-2 pt-0 px-4 pb-px tablet:border-b-0 text-left"
+              className="mt-2 pt-0 px-4 pb-px tablet:border-b-0 text-start"
```

Comment:
It won't let me add a note to L173 in this file and in fxa-payments-server/src/routes/Product/SubscriptionUpgrade/PlanUpgradeDetails.tsx L112, can you change both lines from <div className="flex"> to <div className="flex gap-4">